### ISSUE TITLE
[Bug] Profiles not saving properly.

### ISSUE THREAD

**Issue by hotluck** [failcoder/Supervillain_UI#137]
I started setting things up on my char before realising I hadn't created it as a general use profile, so I created one and finished all my settings and moving the frames. I then exited the game (not logout). When I went back into the char later, it had saved everything before I named the profile but nothing afterwards. I re-did the missing settings and re-moved the frames and this time did a logout. I logged back in and it had saved everything this time. Not sure whether exiting is the problem or whether it is a red herring.

**Comment by hotluck** [comment]
Not sure if I should log this as a separate issue or not but it is still to do with profile saving.

I only use 2 profiles; Level 100 & Non Level 100.  Yesterday when I logged this, I had only created Level 100. Today I created the latter profile. To do this, I selected Share of Level 100 on my character. Then I created Non Level 100 which showed now under the Copy and Share options. I then changed the bottom stats (the only change is to add the EXP and EXP Bar). I logged out at that point and logged back onto a char that had the Level 100 profile. The EXP stats were showing. I decided to set up another char and selected Non Level 100 under Share. It showed the stats that should have been on the Level 100 profile.

It seems that the changes made to the new profile got saved to the original profile. Hope this makes sense :)

**Comment by failcoder** [comment]
I will check into this. 
FYI: t-shirt on it's way....

**Comment by hotluck** [comment]
OMG, talk about a coincidence. I had just opened this and read your comment as there was a knock at the door and my other half shouted that there was a t-shirt here for me. It is awesome...thank you :D

**Comment by failcoder** [comment]
Sorry that it is not the cool blue-ish one that I showed you. The previous screen printing company I used won't make them anymore because they think Blizzard owns my logo. This was also the reason it took so dang long. Did you get the stickers too?

**Comment by hotluck** [comment]
Yes I got the stickers ty and I have to say I prefer the grey tshirt anyway :D
